docs/source/miniconda.rst

```diff
@@ -37,7 +37,7 @@ MacOSX installers
    :widths: 5, 10, 5, 80
 
    Python 3.9,`Miniconda3 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-MacOSX-x86_64.sh>`_,42.2 MiB,``b3bf77cbb81ee235ec6858146a2a84d20f8ecdeb614678030c39baacb5acbed1``
-   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-MacOSX-x86_64.pkg>`_,49.7 MiB,``b7b1628770656361c8b7fb367b30d17e``
+   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-MacOSX-x86_64.pkg>`_,49.7 MiB,``298ff80803817921a98e21d81d60f93b44afce67aec8ae492d289b13741bcffe``
```

Comment:
Yup, that's what I get when I download it. (Sums were generated on Zeus, so just making sure they're the same as what is fetched.)